README.md

```diff
@@ -36,6 +36,32 @@ const AppContainer = () => (
 )
 ```
 
+# Installation
+
+To get started with a local installation run the following:
+
+```bash
+git clone https://github.com/hixme/hixme-ui.git
+cd hixme-ui
+lerna boostrap
+```
+
+Keep in mind [lerna](https://github.com/lerna/lerna) must be installed globally: 
+
+```bash
+npm install -g lerna
+```
+
+To start the documentation React app, run `npm start` inside the `/hui-pages` directory.
+
+## Troubleshooting
+
+Occasionally -- on a fresh install -- `lerna bootstrap` will fail because of a missing `react` dependency. This can be rectified by installing, but not saving the react dependency.
```

Comment:
I don't know if anyone else has run into this, but I figured I'd document it for good measure.

Reply:
I haven't. good to know

Reply:
Should this go on the HUI contribution docs?

Reply:
Definitely. Didn't know we had that.